### DISCUSSION TITLE
airflow task duration calculate mistake when use reschedule mode

### DISCUSSION THREAD

**Discussion by yh2388** [apache/airflow#27466] [Q&A]
version: v2.3.4

Sensor use reschedule mode;

As you can see, according to the log information, start_date is not the time when the real task started, but the time of the last reschedule, so the calculation of duration is wrong

<img width="772" alt="image" src="https://user-images.githubusercontent.com/20351485/199461689-f7c65f0e-7878-4f58-8fd3-1766640e3e18.png">
<img width="1617" alt="image" src="https://user-images.githubusercontent.com/20351485/199459258-02c626a9-e040-4c6c-b711-2b5f7917b72f.png">
<img width="1315" alt="image" src="https://user-images.githubusercontent.com/20351485/199459880-7c9dd244-7ab1-423d-b298-80204362aaa6.png">

**Comment by potiuk** [comment]
You need to open issue for that. not discussion.
